troubleshooting/known_issues/cray_console_node_in_CLBO_state.md

```diff
@@ -0,0 +1,88 @@
+# `CRAY-CONSOLE-NODE` pods intermittently goes into CLBO state after fresh CSM install
+
+## Issue Description
+
+After installing CSM 1.6, `CRAY-CONSOLE-NODE` pods intermittently goes into CLBO state. This happens because of
+permission issue with /var/log/conman directory inside the pods/container.
+
+## Error Identification
+
+When the issue occurs, the `CRAY-CONSOLE-NODE` pods will be in CLBO state. The logs from the pod will be throwing errors as follows:
+
+```text
+2023/02/03 16:30:37 Starting a new instance of command
+2023/02/03 16:30:37 Starting to parse file: /var/log/conman/console.x3000c0s7b0n0
+2023/02/03 16:30:37 Starting to parse file: /var/log/conman/console.x3000c0s21b3n0
+2023/02/03 16:30:37 Starting command process
+2023/02/03 16:30:37 Starting to parse file: /var/log/conman/console.x3000c0s9b0n0
+panic: runtime error: invalid memory address or nil pointer dereference
+[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x701979]
+
+goroutine 57 [running]:
+main.watchConsoleLogFile({0x835878, 0xc0003185c0}, {0xc000040920, 0xd})
+    /usr/local/golib/src/console_node/logAggregation.go:154 +0x2b9
+created by main.aggregateFile
+    /usr/local/golib/src/console_node/logAggregation.go:68 +0x14b
+```
+
+## Fix Description
+
+The workaround is to assign the correct permissions to the /var/log/conman directory inside the pods/container.
+
+1. (`ncn-mw#`) Find the `cray-console-node` pod IDs.
+
+    ```bash
+    kubectl get pods -n services --no-headers -o wide | grep cray-console-node | awk '{print $1}'
+    ```
+
+   Example output:
+
+    ```text
+    cray-console-node-0
+    cray-console-node-1
+    ```
+
+1. (`ncn-mw#`) Log into one of the `cray-console-node` pods using their IDs.
+
+    ```bash
+    kubectl exec -n services -it CRAY-CONSOLE-NODE-POD-ID -- /bin/sh
+    ```
+
+1. Change the permission of the /var/log/ directory.
+
+    ```bash
+    chmod -R 700 /var/log/
+    ```
+
+1. Verify the permission of the /var/log/ directory.
+
+    ```bash
+    ls -ld /var/log/
+    ```
+
+    Example output:
+
+    ```text
+    drwx------ 2 nobody nobody 15 Jan 30 22:05 conman
+    drwx------ 2 nobody nobody 21 Jan 30 22:05 conman.old
+    drwx------ 2 nobody nobody  1 Dec  1  2023 console
```

Comment:
With the ls arguments you used, I think the output would be:
```
drwx------ 5 nobody nobody 3 Jan 20 13:20 /var/log/
```

I think if you want to get the output you are showing, you'd have to run
```
ls -ld /var/log/*
```